databases/week3/Additional_queries.sql

```diff
@@ -0,0 +1,49 @@
+
+-- Get meals that has a price smaller than a specific price fx 90
+use mealsharing;
+SELECT * from meal
+where price  < 90;
+
+-- Get meals that still has available reservations
```

Comment:
A few things:
 1. the query crashes for me;
 2. how are you selecting only meals that still have available reservations? I don't see a `WHERE` clause;
 3. what about meals without reservations? (hint: type of join)

And once you fix number 2., you might want to look at this function COALESCE(): https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_coalesce